.github/workflows/deploy-godot-editor-release.yaml

```diff
@@ -146,6 +154,17 @@ jobs:
           asset_name: godot-kotlin-jvm_editor_${{ matrix.platform }}_${{ steps.save_release_info.outputs.godot_kotlin_jvm_version }}.zip
           asset_content_type: application/zip
 
+      - name: Upload zip release
+        if: matrix.platform == 'server'
+        uses: actions/upload-release-asset@v1.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.save_release_info.outputs.upload_url }}
+          asset_path: godot-kotlin-jvm_editor_headless_${{ steps.save_release_info.outputs.godot_kotlin_jvm_version }}.zip
+          asset_name: godot-kotlin-jvm_editor_headless_${{ matrix.platform }}_${{ steps.save_release_info.outputs.godot_kotlin_jvm_version }}.zip
```

Comment:
I don't think creating an extra step only for adding headless is a good idea, as we already have `server` as platform, which already give information.

Reply:
But how do we distinguish between server and headless editor builds then?
Because on the release page we would then have three builds named server hard to distinguish which one is the headless editor and which ones are the server builds

Reply:
Will sever have `editor` in its release name ? Because here name starts with `godot-kotlin-jvm_editor`

Reply:
Not the servers, but the headless editor as it's technically still the editor.
I could rename that into just `headless_editor`? Maybe it's more clear then that that one is the headless version of the editor?

Reply:
What I was meaning is that you don't need to call it editor headless server, editor server is sufficient, this way you do not need two different steps, and make the CI file easier to maintain.

Reply:
Ah now i get it. Yeah true. I'll change it back.

Reply:
Updated url: https://github.com/chippmann/godot-kotlin-jvm/releases/tag/0.5.3-3.5.1-SNAPSHOT